pipelines/tekton/gitops-update-pipeline/gitea-edge-user-1-secret.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: gitops-git-user-basic-auth
+  name: gitea-edge-user-1
```

Comment:
What if the end user wants to use something other than gitea? Wouldn't keeping it generic not be better/less confusing?

Reply:
I prefer the more specific name here, because the point of it is for this Secret to bring the specifics of the user details on the particular git server to the generic pipeline. That is to say, it's more tied to the git server & user than to the PipelineRun. _Hypothetically,_ there might be many such secrets that are for different Git servers & different users on them, all of which might be used with the generic pipeline in different scenarios. In such a case, generic naming of the secrets wouldn't help. WDYT?

Reply:
When you put it like that I totally agree, good argument 👍. This may be a little pedantic but would you mind including a note somewhere that the user isn't limited to gitea anyway? I think I would accidentally assume that if it was me.

Reply:
Added some words to the README, thanks.